collections/blog.item

```diff
@@ -52,6 +52,31 @@
           {.end}
         </ul>
       {.end}
+      <div class="social-links">
+        <div class="social-links__title">Share this article</div>
+        <ul class="social-links__icons">
+            <li>
+              <a
+                 href="http://twitter.com/share?url={permalink}"
+                 title="Red Badger Twitter"
```

Comment:
This is a bit misleading - should be something like "Share this article on Twitter / Facebook"